kukur/cli.py

```diff
@@ -170,9 +171,13 @@ def _api_keys(kukur_app: Kukur, args):
             writer.writerow((key.name, key.creation_date.isoformat()))
 
 
-def _run():
+def run():
+    """Start Kukur Cli.
+
+    This reads the configuration in Kukur.toml."""
     args = parse_args()
     config = from_toml(args.config_file)
+    configure(config, threaded=False)
```

Comment:
`configure()` is pretty ambiguous. Either import it as `configure_logging()` or just use the fully qualified name.